src/Runner.Worker/ExecutionContext.cs

```diff
@@ -63,6 +63,8 @@ public interface IExecutionContext : IRunnerService
         // Keep track of embedded steps states
         Dictionary<Guid, Dictionary<string, string>> EmbeddedIntraActionState { get; }
 
+        IList<Issue> EmbeddedIssues { get; }
```

Comment:
What about making all ExecutionContexts within the composite action share the same List, and letting the root one add these issues to the timeline record when we call `ExecutionContext.Complete()`?

I prefer to only make changes to the `ExecutionContext.cs` if possible.

Reply:
The overall root ExecutionContext?  Or the more localized root of the composite action?

Reply:
Imo i would do it on complete per the root of the composite action, not **the** root execution context